### PR TITLE
[16.0][FIX] web_timeline: Fixed error message when saving and closing form view in timeline #2619

### PR DESCRIPTION
**Pull request by baguenth**
Fixes the critical issue identified in https://github.com/OCA/web/issues/2619

CC @CRogos @chienandalu @pedrobaeza

### CODE REVIEW

File: web_timeline/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {
     "name": "Web timeline",
     "summary": "Interactive visualization chart to show events in time",
-    "version": "16.0.1.1.5",
+    "version": "16.0.1.1.6",
```
Comment (pedrobaeza) [inline web_timeline/__manifest__.py]: Please don't change manually the module version.

Comment (baguenth) [reply web_timeline/__manifest__.py]: My bad. Just removed it :)

**Review by pedrobaeza — APPROVED** [review]

**Review by flotho — APPROVED** [review]
LGTM

**Pull request merged**